core/sup_tree_core/system_metrics_reporter.ex

```diff
@@ -76,4 +79,17 @@ defmodule AntikytheraCore.SystemMetricsReporter do
       end)
     Enum.map(absolute_values ++ cumulative_values, fn {label, value} -> {label, :gauge, value} end)
   end
+
+  defp log_too_many_messages_processes() do
+    ps = :recon.proc_count(:message_queue_len, 5)
+    [{_pid, top_len, _info}|_] = ps
```

Comment:
```suggestion
    [{_pid, top_len, _info} | _] = ps
```

To be consistent with the existing code.